_layouts/media.html

```diff
@@ -128,6 +132,17 @@ <h3>Spanish</h3>
 		$(selector).html(html_result);
 };
 
+var tips = [
+  { id: "RkCKG-sruKc", title: "The EASIEST Git workflow guide you'll ever watch", author: "Waldirio M Pinheiro" },
+  { id: "rtTw6xqFtIE", title: "Setting Up the Foreman and Katello Development Environment: A Step-by-Step Guide", author: "Waldirio M Pinheiro" },
+  { id: "SS0vvQFygvM", title: "How to Set Up VScode Icons Extension (2024)", author: "Waldirio M Pinheiro" },
+  { id: "Dyjy85MfFOo", title: "Setting Up The Best VIM Extension for VScode (2024)", author: "Waldirio M Pinheiro" },
+  { id: "LZbi9YwkQ-0", title: "How to Set Up Trailing Spaces Extension in VS Code", author: "Waldirio M Pinheiro" },
+  { id: "b-T14EKqSvI", title: "Setting up Remote SSH in VScode is *really* this easy", author: "Waldirio M Pinheiro" },
+  { id: "V5hVEr6Nt8Q", title: " How to Use Git via VSCode Like a Pro! ", author: "Waldirio M Pinheiro" }
```

Comment:
```suggestion
  { id: "V5hVEr6Nt8Q", title: "How to Use Git via VSCode Like a Pro!", author: "Waldirio M Pinheiro" }
```

Reply:
Hello @ekohl 

Thank you for that, small detail that skipped my eyes.

Fixed.